tests/sentry/deletions/test_group.py

```diff
@@ -195,11 +196,30 @@ def test_delete_groups_delete_grouping_records_by_hash(
 class DeleteIssuePlatformTest(TestCase, SnubaTestCase, OccurrenceTestMixin):
     def test_issue_platform(self):
         event = self.store_event(data={}, project_id=self.project.id)
-        node_id = Event.generate_node_id(event.project_id, event.event_id)
-        self.build_occurrence(event_id=event.event_id)
+        issue_occurrence, group_info = self.process_occurrence(
+            event_id=event.event_id,
+            project_id=self.project.id,
+            # We are using ReplayDeadClickType as a representative of Issue Platform
+            type=ReplayDeadClickType.type_id,
+            event_data={
+                "fingerprint": ["issue-platform-group"],
+                "timestamp": before_now(minutes=1).isoformat(),
+            },
+        )
+        assert group_info is not None
+        issue_platform_group = group_info.group
+        assert len(Group.objects.all()) == 2
```

Comment:
I don't think we need to check the size of the group table here. Maybe check that `event.group_id != issue_platform_group.id` instead?